library/hawkular_alerts_group_trigger.py

```diff
@@ -237,7 +239,7 @@ def update_group_trigger(self, trigger, updates, conditions):
         except Exception as e:
             self.module.fail_json(msg="Failed to update group trigger. Error: {error}".format(error=e))
         if conditions:
```

Comment:
Hmm.  I think this should be `if conditions is not None` as on L290.
If it's explicitly given as empty list, we do want to set_group_trigger_conditions().